### DISCUSSION TITLE
Inspired

### DISCUSSION THREAD

**Discussion by Techno-Fox** [GitSquared/edex-ui#1117] [Questions]
I've been inspired by edex-ui to create a dashboard of sorts. I plan on creating it for cybersec purposes. I'm still really new to webdesign, and have some questions. 

1). Where did you get your audio assets?
2). Any tips and tricks on web development?
3). Any recommended classes/libraries?

**Comment by 3203234030** [comment]
I don' t know , what You intended. .

**Reply by Techno-Fox** [reply]
For a while i noticed that y'all really like javascript, so I was asking if y'all knew any tips and tricks for web design in general. I just started learning javascript (I'm used to lower level languages). Web design is something I've done before, but only for class standards. Most of my programs run in a terminal. Not bad, but would like to create something better. I actually tried electron with golang. Didn't go so well. So i decided to just go raw web dev. However I was also interested in where the audio assets came from. If they made them themselves or if they acquired them from other means.

**Reply by GitSquared** [reply]
Don't mind him, just lost in notifications

**Comment by GitSquared** [comment]
Glad you liked it!

The audio assets were custom-made by a friend of mine, as the [README mentions](https://github.com/GitSquared/edex-ui#credits).

I'm not sure what background/skill level you're coming from, so I don't really know what to tell you about web dev. If you're looking to learn, most of the work I do professionally (and see people working with) builds on top of React, so you might want to check it out.

**Reply by Techno-Fox** [reply]
React. I've heard of it, but never really looked into it.

**Reply by Techno-Fox** [reply]
You made all the css, and page design by hand?

**Reply by GitSquared** [reply]
> You made all the css, and page design by hand?

Yup.
